src/main/features/response/guards/countyCourtJudgmentRequestedGuard.ts

```diff
@@ -0,0 +1,17 @@
+import * as express from 'express'
```

Comment:
this is unexpected...
This functionality is scheduled for next sprint in a story which needs some design work:
https://tools.hmcts.net/jira/browse/ROC-2485

Reply:
Worth checking with business how not having it impacts planned release. I do not mind this temporary solution in place if that would let us make feature available to the users sooner.

Reply:
Clement Neveu [4:06 PM] 
@Tim I think we should be able to release without it: The judgment will be recorded on Caseman and the support team will therefore not process the Defence and send it out to the claimant (edited)

Reply:
However defence is then recorded on the case which puts it in inconsistent state. I would pretty much prefer to avoid it, despite the fact risk might not be huge.